source/_integrations/tod.markdown

```diff
@@ -6,20 +6,34 @@ ha_category:
 ha_release: 0.89
 ha_iot_class: Local Push
 ha_quality_scale: internal
+ha_config_flow: true
 ha_domain: tod
 ha_platforms:
   - binary_sensor
 ---
 
-The `tod` platform supports binary sensors which get their values by checking if the current time is within defined time ranges.
+The Times of the Day integration provides a binary sensor which get their values by checking if the current time is within defined time ranges.
```

Comment:
Choose one of:
- *provides a binary sensor which gets its value*
- *provides binary sensors which get their values*

Reply:
Almost... `which` -> `that` :) 

Thanks for spotting that one 👍 
```suggestion
The Times of the Day integration provides a binary sensor that gets its values by checking if the current time is within defined time ranges.
```